### PR TITLE
MuhammedKalkan.OpenLens version 6.2.5

### PR DESCRIPTION
**Pull request by brotich**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90946)

### CODE REVIEW

File: manifests/m/MuhammedKalkan/OpenLens/6.2.5/MuhammedKalkan.OpenLens.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+PackageIdentifier: MuhammedKalkan.OpenLens
+PackageVersion: 6.2.5
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/m/MuhammedKalkan/OpenLens/6.2.5/MuhammedKalkan.OpenLens.installer.yaml:13]:
```suggestion
- Architecture: x64
```

File: manifests/m/MuhammedKalkan/OpenLens/6.2.5/MuhammedKalkan.OpenLens.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://github.com/MuhammedKalkan/OpenLens/releases/download/v6.2.5/OpenLens-6.2.5.exe
+  InstallerSha256: 801CD4FB6782FBFEB2813F2519BB7F585D5B462B8649C317E55300128C0EC7CD
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/m/MuhammedKalkan/OpenLens/6.2.5/MuhammedKalkan.OpenLens.installer.yaml:20]:
```suggestion
- Architecture: x64
```

Comment (stephengillie) [reply manifests/m/MuhammedKalkan/OpenLens/6.2.5/MuhammedKalkan.OpenLens.installer.yaml:20]: Packages marked x86 appear to be validated with a 32-bit operating system.

**Comment by brotich** [comment]
@stephengillie i have no idea how i missed the arch. Thanks for checking

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**